packages/core/src/tracing/cls.js

```diff
@@ -48,6 +50,7 @@ function init(config, _processIdentityProvider) {
     serviceName = config.serviceName;
   }
   processIdentityProvider = _processIdentityProvider;
+  allowRootExitSpan = process.env.INSTANA_ALLOW_ROOT_EXIT_SPAN === '1' || config?.tracing?.allowRootExitSpan;
```

Comment:
issue: If we normalize the configuration options, there is no need to check for the env variable here again.